README.md

```diff
@@ -92,14 +92,11 @@ If this is your first time you are using Vagrant, you will need:
   [Homebrew](http://brew.sh), but it's not required. Windows users can use the
   excellent [Babun](http://babun.github.io) or get Git directly from
   [git-scm.com](https://git-scm.com).
-* [VirtualBox](https://www.virtualbox.org), used to run virtual machines.
+* [VirtualBox](https://www.virtualbox.org), used to run virtual machines. Use
+  the latest version instead of distribution-provided packages.
```

Comment:
How does one find the latest version vs `distribution-provided` packages? Are you saying go to the website instead of using things like apt-get install or yum install?

Reply:
Both Virtualbox and Vagrant provide debian packages or apt repositories. For example, Ubuntu 16.04 ships with vagrant 1.9.1, which can no longer download boxes (Hashicorp changed the hostname) and it's unlikely to ever be backported since they are in universe and not main.

https://www.vagrantup.com/downloads.html

https://www.virtualbox.org/wiki/Linux_Downloads

Reply:
So this probably makes sense for someone installing from Linux, but for someone installing from Windows or Mac, will they get confused and wonder if they are using the correct version? Would this be clearer?

> Make sure you are installing the latest version from the website.

Reply:
I like it, done!